time stamp 
nav_msgs/OccupancyGrid map
RoadMapNode[] nodes
RoadMapEdge[] edges